src/lib/operators/join_hash/join_hash_steps.hpp

```diff
@@ -136,22 +136,23 @@ class PosHashTable {
   // Rewrite the SmallPosLists into one giant UnifiedPosList (see above).
   void finalize() {
     _offset_hash_table.shrink_to_fit();
+    const auto hash_table_count = _offset_hash_table.size();
```

Comment:
Minor: `hash_table_count` to me sounds like a count of multiple hash tables, not like the size/number of entries in a single hash table.

Reply:
Good catch.